README.md

```diff
@@ -379,9 +380,15 @@ Uncertainty predictions may be calibrated to improve their performance on new pr
 * `mve_weighting` For use with ensembles of models trained with mve or evidential loss function. Uses a weighted average of the predicted variances to achieve a minimum negative log likelihood of predictions. (https://doi.org/10.1186/s13321-021-00551-x)
+* `conformal_regression` Generates an interval of fixed size for each prediction such that the actual value has probability $1-\alpha$ of falling in the interval. The desired error rate is controlled using the parameter `--alpha <float>` which is set by default to 0.1. (https://arxiv.org/abs/2107.07511)
+* `conformal_quantile_regression` Generates an interval for each prediction based on quantile regressions of the data. The desired error rate is controlled using the parameter `--alpha <float>` which is set by default to 0.1. The loaded model should output quantile predictions. (The trained model should output $\alpha/2$ and $1-\alpha/2$ quantiles.) Such a model model can be trained with parameters `--loss_function quantile_interval` and `--alpha <float>` where $\alpha$ is the desired error rate. (https://arxiv.org/abs/2107.07511)
+
 **Classification**
 * `platt` Uses a linear scaling before the sigmoid function in prediction to minimize the negative log likelihood of the predictions. If the model checkpoint was generated after Chemprop v1.5.0, then a Bayesian correction is applied to account for the class balance in the training set during prediction. Implemented for classification but not multiclass datasets. (https://arxiv.org/abs/1706.04599)
 * `isotonic` Fits an isotonic regression model to the predictions. Prediction outputs are transformed using a stepped histogram-style to match the empirical probability observed in the calibration data. Number and size of the histogram bins are procedurally decided. Histogram bins are wider in the regions of the model output that are less reliable in ordering confidence. Implemented for both classification and multiclass datasets. (https://arxiv.org/abs/1706.04599)
+* `conformal` Generates a set of possible classes for each prediction such that the true class has probability 1-alpha of falling in the set. The user can control the desired error rate using the parameter `--alpha <float>` which is set by default to 0.1. Set generated using the basic conformal method. (https://arxiv.org/abs/2107.07511)
+* `conformal_adaptive` Generates a set of possible classes for each prediction such that the true class has probability 1-alpha of falling in the set. The user can control the desired error rate using the parameter `--alpha <float>` which is set by default to 0.1. Set generated using the adaptive conformal method. (https://arxiv.org/abs/2107.07511)
```

Comment:
Are the conformal and conformal_adaptive methods only used for multiclass predictions?

Reply:
Hey Charles, I actually forgot to separate these 2 methods into a different section. The conformal method can be used for both multiclass and classification. However since the methods are sufficiently different, I separated the methods into a different section for multiclass and classification.

About the naming, the names are really similar because the methods are similar was well. I am using the standard names from the paper that is linked. Do you think that is ok?